lib/views/partials/scripts.hbs

```diff
@@ -15,5 +15,7 @@
 <script src="./js/views/search-view.js"></script>
 <script src="./js/views/extracurricular-view.js"></script>
 
```

Comment:
nit: can we group views together (and other backbone components alike) together, and name views as `views/**-view.js`
```
<script src="./js/views/search-view.js"></script>
<script src="./js/views/extracurricular-view.js"></script>
<script src="./js/views/course-display-view.js"></script>
```